assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -532,3 +532,6 @@ che.server.secure_exposer.jwtproxy.token.ttl=8800h
 che.server.secure_exposer.jwtproxy.auth.loader.path=/_app/loader.html
 che.server.secure_exposer.jwtproxy.image=eclipse/che-jwtproxy:latest
 che.server.secure_exposer.jwtproxy.memory_limit=128mb
+
+# Maximum size of the json processing pool
+che.server.jsonrpc.processor_max_pool_size=100
```

Comment:
I don't think that this property is related to Che Server. Please consider renaming it to something more common like `che.core.jsonrpc...` or `che.jsonrpc....` . Also, it makes sense to place it near WebSocket related properties.